### ISSUE TITLE
Include JSON-LD specifications as `git` submodules

### ISSUE THREAD

**Issue by anatoly-scherbakov** [digitalbazaar/pyld#181]

**Comment by BigBlueHat** [comment]
My one concern here is that git submodules need to be updated if/when the upstream repo changes as they're pinned to specific commit SHAs. It may be all that's needed is a simple python script to run the commit commands needed to pull the latest repos locally.

**Comment by anatoly-scherbakov** [comment]
@BigBlueHat Indeed, periodically one has to update the submodules. I would argue this is a good thing, similar to how one has to explicitly update Python dependencies of a project: it makes the project more reliable, and the builds more stable.

Updating dependencies, including the specs, should be a conscious decision and a human should be in the loop to fix possible issues.

Otherwise, a test run for a completely unrelated change can throw a bunch of errors clearly unrelated to the change, and it will take time to realize that one of the specs (and which one?) has been meanwhile updated.
